css/spica-styles.scss

```diff
@@ -0,0 +1,54 @@
+#contactsmenu {
+	margin-right: 1.5rem;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.nmc_spica_wrapper {
+	.icon-mail {
+		background-image: var(--icon-mail-custom) !important;
+	}
+	.has-unread {
+		margin-top: -20px;
+		margin-left: -18px;
+		color: var(--white-color) !important;
+		font-weight: 700 !important;
+		height: 0.875rem !important;
```

Comment:
not required to add important... as we can modify original styling ...
and also you can reduce styling by changing original one